src/js/views/CertificationAuthorities/layout/style.js

```diff
@@ -95,4 +95,9 @@ export const useDataTableStyles = makeStyles(theme => ({
   tooltip: {
     fontSize: '1rem',
   },
+  fingerprintCell: {
+    display: 'flex',
+    alignItems: 'center',
+    height: 61,
```

Comment:
I believe that setting the height this way is a bad practice. Maybe if you add a `<div>` inside the table cell the `display:flex` would not be a problem anymore.